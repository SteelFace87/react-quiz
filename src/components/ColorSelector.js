import React from 'react';
import PropTypes from 'prop-types';

export default function ColorOptions({ handleColorChange }){
  return (
    <>
    <h2>Select color for box/text below</h2>
    <select onChange={handleColorChange}>
      <option value='green'>green</option>
      <option value='blue'>blue</option>
    </select>
    </>
  );
}

ColorOptions.propTypes = {
  handleColorChange: PropTypes.func.isRequired
}
;
