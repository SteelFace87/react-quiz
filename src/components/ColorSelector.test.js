import React from 'react';
import { shallow } from 'enzyme';
import ColorSelector from './ColorSelector';

describe('Shape', ()=>{
  it('renders stuff', ()=>{
    const wrapper =  shallow(<ColorSelector handleColorChange={()=>'work!'}/>);
    expect(wrapper).toMachSnapshot;
  });
})
;
