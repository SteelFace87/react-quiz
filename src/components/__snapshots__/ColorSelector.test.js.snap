// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape renders stuff 1`] = `
<Fragment>
  <h2>
    Select color for box/text below
  </h2>
  <select
    onChange={[Function]}
  >
    <option
      value="green"
    >
      green
    </option>
    <option
      value="blue"
    >
      blue
    </option>
  </select>
</Fragment>
`;
